pulpcore/pulpcore/app/serializers/status.py

```diff
@@ -48,8 +48,13 @@ class StatusSerializer(serializers.Serializer):
         many=True
     )
 
-    known_workers = WorkerSerializer(
-        help_text=_("List of celery workers known to the application"),
+    online_workers = WorkerSerializer(
+        help_text=_("List of online celery workers known to the application"),
```

Comment:
What about expanding this out a bit more and let users know what 'online' means?